_meta/fields.common.yml

```diff
@@ -685,3 +685,80 @@
       dynamic: true
       description: Additional experimental data sent by the agents.
 
+    - name: cloud
+      title: Cloud
+      group: 2
+      type: group
+      description: >
+        Cloud metadata reported by agents
+      fields:
+      - name: account
+        type: group
+        dynamic: false
+        fields:
+        - name: id
+          level: extended
+          type: keyword
+          ignore_above: 1024
+          description: Cloud account ID
+        - name: name
+          level: extended
+          type: keyword
+          ignore_above: 1024
+          description: Cloud account name
+      - name: availability_zone
```

Comment:
I was expecting these to need `overwrite: true` - that doesn't seem to be enforced any more - this is worrying.

Reply:
So apparently it fails when you run `apm-server export index-pattern`. I'll fix this instance and create an issue to make sure this gets caught in CI.

Reply:
I've updated this file, and opened #3740 to fix the systemic problem.